src/utils/self_test/SConscript

```diff
@@ -9,24 +9,40 @@ import os
 import daos_build
 
 SELF_TEST = 'self_test.c'
+LIB_CONFIG_INI = 'src/configini.c'
+LIB_CONFIG_CNF = 'etc/selftest.cnf'
+LIB_CONFIG_ECNF = 'etc/selftest_expected.cnf'
+
+# Function to filter out gcc flags from this compilation env
+def remove_certain_gcc_flags(str):
+    import re
+    r = re.compile("\-Wframe\-larger\-than\=")
```

Comment:
(lint) '\-'. String constant might be missing an r prefix.
(lint) '\-'. String constant might be missing an r prefix.
(lint) '\-'. String constant might be missing an r prefix.
(lint) '\='. String constant might be missing an r prefix.